cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -438,7 +440,14 @@ public KafkaAssemblyOperator(Vertx vertx, boolean isOpenShift,
         }
 
         Future<ReconciliationState> zkStatefulSet() {
-            return withZkDiff(zkSetOperations.reconcile(namespace, zkCluster.getName(), zkCluster.generateStatefulSet(isOpenShift)));
+
+            Future<Secret> secretFuture = secretOperations.getAsync(namespace, KafkaResources.clusterCaCertificateSecretName(name));
```

Comment:
`Ca.caCertSecret()` should already have the `Secret` available.

Reply:
That's right! Damn!